install.bash

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [[ "$1" == "inchina" ]]; then
+  export WE_ARE_IN_CHINA=true
+fi
+
+echo
+echo "Welcome to SQLFlow playground!"
+echo
+
+if ! which vagrant >/dev/null; then
+  echo "Install vagrant first, please refer to https://www.vagrantup.com/downloads.html"
+  echo
+  echo "If you are using macOS, the operation system may prevent you from installing the web-downloaded package, you can try use brew to install vagrant. like:"
+  echo "brew cask install vagrant"
+  echo
+  echo "Fix this and re-run this script again please!"
+  exit 0
+fi
+
+if [[ -n "$(vagrant global-status --prune | grep -o 'playground')" ]]; then
+  echo "It seems you have already installed our playground, exiting..."
+  exit 0
+fi
+
+if [[ "$WE_ARE_IN_CHINA" ]]; then
+  if [[ -z "$(vagrant box list | grep -o ubuntu/bionic64)" ]]; then
+    echo "Download ubuntu box beforehand..."
+    mkdir -p downloads
+    # try with https://mirrors.ustc.edu.cn/ if below not work
+    wget -c -nv --show-progress -O downloads/ubuntu-bionic64.box "https://mirrors.ustc.edu.cn/ubuntu-cloud-images/bionic/current/bionic-server-cloudimg-amd64-vagrant.box"
```

Comment:
This script might run on macOS, Windows, and Linux, right?  It needs the way to install axel.

- macOS: 
  1. install Homebrew
  1. install axel, VirtualBox, and Vagrant using Homebrew
- Windows:
  1. install Chocolatey
  1. install axel, VirtualBox, and Vagrant using Chocolatey
- Debian-like Linux:
  1. update apt-get
  1. install axel, VirtualBox, and Vagrant using apt-get

Reply:
Like described above, now, I'm focused on macOS and Linux, especially for Chinese developers. Currently, make the provision progress fast takes a higher priority. Maybe I can fix this up in future PRs.